### ISSUE TITLE
DateRangePicker: translate "Custom Range" label

### ISSUE THREAD

**Issue by mblint-postcron** [valor-software/ng2-bootstrap#5768]
**Description:**
I'm using locales settings to change date picker language and it works like a charm. The only thing is not translating is the "Custom Range" label when you add predefines custom ranges to range picker.

I couldn't find any other config to change that label.

Thanks,
Mauricio.

**Comment by niklas-dahl** [comment]
I am facing the same problem, my current solution until this gets fixed is to just not show the button.

```
// there is no translation for 'Custom Range' string :(
.bs-datepicker-predefined-btns .btn:last-child {
  display: none;
}
```

**Comment by rafaelss95** [comment]
Should be solved by #5918.

**Comment by jorgenlaaker** [comment]
When we use jquery, another solution is:  $('[data-range-key="Custom Range"]').text('Translated text');
